### PR TITLE
Added some quick tweaks

### PR DESCRIPTION
**Pull request by jurvis**
A brief list:
1. use nav/main/footer to organise content
2. add headers and subheaders into a hero container
3. added a new media query variable `mq-not-tablet`
4. added some padding rules for desktop/tablet views
5. use em unit in favour of px
6. change interaction of buttons

### CODE REVIEW

**Comment by jurvis** [comment]
@duncanleo please have a look and feel free to refactor! thanks 😄

File: .gitignore [hunk 1]
```diff
@@ -87,3 +87,6 @@ typings/
 # End of https://www.gitignore.io/api/node
 
 .DS_Store
+
+
+dist/*
```
Comment (duncanleo) [inline .gitignore]: @jurvis we need to allow the dist folder to be committed - there is a `_redirects` file in there necessary for Netlify.
```suggestion
```

Comment (duncanleo) [reply .gitignore]: I believe the `.gitignore` currently in `dist` should be sufficient, but let me know if it is better to have it here

Comment (duncanleo) [reply .gitignore]: Okay, never mind. I realised that the `.gitignore` in `dist/` was not committed properly.

**Review by duncanleo — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jurvis) [reply .gitignore]: @duncanleo can you please commit it so I can remove this change and rebase without issues?

Comment (duncanleo) [reply .gitignore]: @jurvis done!

**Pull request merged**